modules/core/src/com/haulmont/cuba/security/auth/AuthenticationManager.java

```diff
@@ -73,4 +73,11 @@
      * @see AuthenticationService#logout()
      */
     void logout();
+
+    /**
+     * Checks if user is active
+     * @param user user to check
+     * @return true if user is active
+     */
+    boolean isUserActive(User user);
```

Comment:
Please move a code from `AuthenticationManager` & `AuthenticationService` to `UserManagementService`